ext/src/http/client/curl/http_client_curl.cc

```diff
@@ -811,6 +830,42 @@ bool HttpClient::doRemoveSessions()
   return has_data;
 }
 
+bool HttpClient::doRetrySessions(bool report_all)
+{
```

Comment:
nit, optional:

Please use ifdef ENABLE_OTLP_RETRY_PREVIEW as well in HttpClient::doRetrySessions(), to protect this code path.